tests/strategy/failing_strategy.py

```diff
@@ -0,0 +1,87 @@
+
```

Comment:
Not necessarily relevant for this PR, but should we move the test-strategies into a dedicated subfolder? 
we now got 2 - but i guess there may be more in the future - and if we move DefaultStrategy here too it might become a bit ... messy ? 

also, i think it would make testing the resolver easier - as now it may also find strategies imported in test files ...

Reply:
> Not necessarily relevant for this PR, but should we move the test-strategies into a dedicated subfolder?

also thought about this.
probably into a dir (not a folder 🤣 ) with other testing data...